### DISCUSSION TITLE
Flow Variables similar to Dynamic variables

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#20935] [Feature Requests]
### Summary

I would like to use something like $NOW

### Basic Example

usage in a payload to set date to now

### Motivation

Improvement of flows

### Detailed Design

-

### Requirements List

Support all dynamic variables

### Drawbacks

support some dynamic variables. Or atleast let "new Date()" be the same as of host browser?

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
